packages/packagers/css/src/CSSPackager.js

```diff
@@ -7,7 +7,23 @@ export default new Packager({
     let promises = [];
     bundle.traverseAssets({
       exit: asset => {
-        promises.push(asset.getCode());
+        let parentDep;
+        bundle.traverseAncestors(asset, (node, _, {stop}) => {
+          if (node.type === 'dependency') {
+            parentDep = node.value;
+            stop();
```

Comment:
The algorithm here appears a little different from what Parcel 1 was doing. Parcel 1 collected all media values from all incoming dependencies and joined them together, whereas this stops after the first dependency. If a dep was seen that didn't have a media setting at all, then it is meant for all media, and so the wrapper is not added. I think perhaps you could use `skipChildren` here instead, and just push the media type to an array.

Reply:
Yea, I was thinking the same thing. 

https://github.com/parcel-bundler/parcel/pull/3106#discussion_r289544286

Since, we are going upward does `skipChildren` make sense?

Reply:
Yeah... I guess the API is a bit confusing. "children" in this case refer to the next items we would traverse (in this case they are parents). Maybe we should reconsider that name, but it should do what you want.

Reply:
Updated! This should be properly going up the chain and stopping at the first none wrapped dependancy